src/content/docs/en/guides/deploy/microsoft-azure.mdx

```diff
@@ -42,3 +42,5 @@ The GitHub action yaml that is created for you assumes the use of node 14. This
 ## Community Resources 
 
 - [Deploying an Astro Website to Azure Static Web Apps](https://www.blueboxes.co.uk/deploying-an-astro-website-to-azure-static-web-apps)
+- [Creating a Static Site with Astro, Azure Static Web Apps, and GitHub](https://agramont.net/blog/create-static-site-astro-azure-ssg/)
+- [Using Microsoft Entra Id EasyAuth with Astro and Azure Static Web App](https://agramont.net/blog/entra-id-easyauth-with-astro/)
```

Comment:
```suggestion
- [Deploying a Static Astro Site to Azure Static Web Apps using GitHub Actions](https://agramont.net/blog/create-static-site-astro-azure-ssg/#automate-deployment-with-github-actions)
```
It seems more helpful to link to the part about deployment specifically, I think!